packages/components/src/Field/Field.stories.mdx

```diff
@@ -1,57 +1,64 @@
-import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
 import { Field } from './Field';
-import { useStyles } from '@marigold/system';
 
-<Meta title="Components/Field" />
+<Meta
+  title="Components/Field"
+  argTypes={{
+    variant: {
+      control: {
+        type: 'text',
+      },
+      table: {
+        defaultValue: {
+          summary: 'default',
+        },
+      },
+    },
+    htmlFor: {
+      control: {
+        type: 'text',
+      },
+      type: { required: true },
+    },
+    label: {
+      control: {
+        type: 'text',
+      },
+      type: { required: true },
+    },
+    error: {
+      control: {
+        type: 'boolean',
```

Comment:
Prop error is type string at the moment and changing some props here has no effect.